docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -72,15 +73,21 @@ and will require access to both the public internet and the Teleport Auth Servic
   $ cd teleport-plugins/access/slack
   $ make
   ```
-  
+
   Place the `teleport-slack` binary into an appropriate location
   within the sytem's `PATH`, e.g., `/usr/local/bin`:
 
-  ```code 
-  $ mv ./build/teleport-slack /usr/local/bin 
+  ```code
+  $ mv ./build/teleport-slack /usr/local/bin
   ```
 
-</TabItem> 
+</TabItem>
+<TabItem label="Helm Chart">
+  ```code
+  helm repo add teleport https://charts.releases.teleport.dev/
+  helm repo update
+  ```
+</TabItem>
 </Tabs>
 
   Make sure the binary is installed:
```

Comment:
The command under "Make sure the binary is installed" won't work for the Helm chart. I think we should either delete it or move it into both the "Download" and "From Source" tabs.